lib/vehicle.rb

```diff
@@ -6,17 +6,22 @@ class Vehicle
               :make,
               :model,
               :engine
+              
+  attr_accessor :registration_date,
```

Comment:
This is a great option for this project to allow vehicle attributes to be editable in the facility class

On the job, however, you'll see that we generally want to avoid attr_accessors  as much as possible to maintain data integrity. if you wanted to avoid attr_accessors *completely*, the option would be to create some "setter" methods in this class called `assign_plate_type` and `assign_registration_date` and the Facility class could call those methods to make  changes to those attributes. The benefit to doing this is adding some controls in those methods (i.e. an if statement that makes sure that the plate_type is ONLY going to be set to the value of :ev, :antique, etc. Or a conditional to make sure the year is only set to an integer with 4 digits). We wouldn't expect you to add those controls into such a method as part of this project's requirements but I'm sharing this info so you know why we encourage that type of assignment rather than attr_accessors! Let me know if you have any questions about this

Again - using attr_accessors here is totally fine and we would never mark you down for using them, but it's good to know why you won't see them out in the world very much!